template/build/webpack.base.conf.js

```diff
@@ -8,6 +8,17 @@ function resolve (dir) {
   return path.join(__dirname, '..', dir)
 }
 
+const lintingRule = {
+  test: /\.(js|vue)$/,
+  loader: 'eslint-loader',
+  enforce: 'pre',
+  include: [resolve('src'), resolve('test')],
+  options: {
+    formatter: require('eslint-friendly-formatter'),
```

Comment:
So this want not a bad idea in hindsight - this will break if the user has decided against using eslint during the project init. Will fix it.